finagle-core/src/main/scala/com/twitter/finagle/tracing/Id.scala

```diff
@@ -84,26 +97,30 @@ object TraceId {
         traceId.flags.setFlag(Flags.SamplingKnown)
     }
 
-    val bytes = new Array[Byte](32)
-    ByteArrays.put64be(bytes, 0, traceId.spanId.toLong)
-    ByteArrays.put64be(bytes, 8, traceId.parentId.toLong)
-    ByteArrays.put64be(bytes, 16, traceId.traceId.toLong)
+    val bytes = new Array[Byte](40)
+    ByteArrays.put64be(bytes, 0, traceId.spanId.self)
```

Comment:
I switched over to my open IntelliJ to start looking at a different area of Finagle, unrelated to this PR, and this file was open.  I remembered that these were previously `toLong`, and here I have changed them to `self`. I wasn't even thinking about the change when I did this, I was just directly referencing the property rather than the helper method .  Should I go ahead and change these (in Id.scala, IdTest.scala) back to `toLong`?

Reply:
I suggest leaving it as toLong for clarity